CHANGELOG.md

```diff
@@ -1,11 +1,31 @@
-# Change Log
+# Change Log  <!-- omit in toc -->
 
-v0.2.0 **Updated README.md to new format** (2020-05-29)
+_A detailed history of project versions and associated release notes._
+
+- [Versioning](#versioning)
+- [Releases](#releases)
+- [History](#history)
+
+## Versioning
+
+This project uses **Semantic Versioning 2.0.0**. Please see [SemVer](https://semver.org/) to read the specification.
+
+## Releases
+
+All released versions are available as [releases on this repository](https://github.com/tforster/git-new/releases).
+
+## History
+
+`v0.3.0` **Improved README and other MD files** (2020-11-20)
+
+- [#0623 Create new README.md template](https://dev.azure.com/techsmarts/TechSmarts/_workitems/edit/623)
```

Comment:
What are your thoughts on linking to a private ADO page from the public repo?  I guess it only helps us two, but not anyone else using this project.

Reply:
My expectation is that the CHANGELOG.md template is going to be updated per project, making the link itself project-specific. I toyed with spelling that concept out in prose but opted to illustrate it literally.